tests/aws/services/kms/test_kms.py

```diff
@@ -107,6 +107,51 @@ def test_create_key_custom_id(self, kms_create_key, aws_client):
         assert result["KeyId"] == key_id
         assert result["Arn"].endswith(f":key/{key_id}")
 
+    @markers.aws.only_localstack
+    def test_create_key_custom_key_material_hmac(self, kms_create_key, aws_client):
+        custom_key_material = base64.b64encode(b"custom test key material").decode("utf-8")
+        key_spec = "HMAC_256"
+        mac_algo = "HMAC_SHA_256"
+        expected_mac = b"}\xa2\x95\xb8\xac\xc9*$\xe7!G\xdb\xf2\xc7:8y\xb1\xcf\xdf\xcc[5\xdcA\xd0e\x98\x13d\x88\x8e"
```

Comment:
Can we give add some more context here, like rather than using a static value we can actually use some encoding method to generate these bytes which can be used by the below section of code.